meson.build

```diff
@@ -163,7 +166,23 @@ if is_parallel_build
   endif
   extra_cmp_args += [ '-D__WITH_MPI__']
   with_mpi = true
+
+  # find netcdf
+  if build_machine.system() != 'windows'
+    netcdf = dependency('netcdf', language : 'fortran', required : false)
+  else
+    # For CI testing only; Windows not yet supported
+    nc_dir = meson.project_source_root() / '..' / 'ncf' / 'netcdf-fortran-4.6.1' / 'fortran'
+    netcdf = fc.find_library('netcdff', dirs: [ nc_dir ], required : true, static : false)
```

Comment:
required should be set false here to avoid a meson abort

Reply:
eventually we need to decide if we are going to allow the `extended` version to cherry pick including petsc or netcdf